.github/workflows/terraform.yaml

```diff
@@ -32,10 +32,11 @@ jobs:
       - name: Terraform Validate
         id: validate
         run: terraform validate -no-color
-      - name: Terraform Plan
-        id: plan
-        run: terraform plan -no-color
-        continue-on-error: true
+# Disable for now, hangs indefinitely
```

Comment:
what step does this hang at? Are we accidentally connecting to TF Cloud? In which case it would wait for the 'workspace' selection indefinitely.